pom.xml

```diff
@@ -1,23 +1,269 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>no.fdk</groupId>
-    <artifactId>fdk</artifactId>
-    <version>1.0</version>
-    <packaging>pom</packaging>
-
-    <name>FDK Reference data</name>
-    <description>Reference-data is a service that provides metadata for the various applications in this ecosystem</description>
-    <organization>
-        <name>Brønnøysundregistrene</name>
-        <url>http://brreg.no</url>
-    </organization>
-
-    <modules>
-        <module>applications/reference-data</module>
-    </modules>
+    <artifactId>reference-data</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.8.RELEASE</version>
+        <relativePath/>
+    </parent>
+
+    <properties>
+        <!--standard properties-->
+        <java.version>1.8</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.exec.skip>false</maven.exec.skip>
+        <!--end standard properties-->
+
+        <start-class>no.dcat.themes.ApplicationThemes</start-class>
+        <docker.image.name>dcatno/reference-data</docker.image.name>
+        <docker.registry/>
+        <unittest.category/>
+        <endtoendtest.category/>
+        <loadtest.category/>
+        <integrationtest.category/>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <!-- må ha actuator for å kjøre på openshift. kreves av readyness probe -->
```

Comment:
Hmmm. Denne kommentaren håpar eg ikkje gjelder lenger. Eller må vi ha det for _kubernetes_?

Reply:
Det er nok bare noe som henger igjen fra gammelt av, har ikke lagt merke til noen konsekvenser av å fjerne det fra andre apper.